### PR TITLE
Added in RTL support to storybook as well as testing function

### PR DESCRIPTION
**Pull request by micahgodbolt**
First pass on supporting RTL in visual regression testing.

### CODE REVIEW

File: apps/vr-tests/src/utilities/steps.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Steps as OriginalSteps } from 'screener-storybook/src/screener';
+
+OriginalSteps.prototype.setRTL = function (isRTL: boolean = false) {
+
+  var step = {
+    type: 'executeScript',
+    code: `document.documentElement.setAttribute("dir", ${isRTL ? '"rtl"' : '"ltr"'})`
```
Comment (MLoughry) [inline apps/vr-tests/src/utilities/steps.ts]: Doesn't `getRTL()` in utilities cache this value?

Comment (dzearing) [reply apps/vr-tests/src/utilities/steps.ts]: yes. If this is a page refresh, then cool. but we can't simply flip and expect things to work without a re-render.

You can call setRTL in utilities to change it and re-render

Comment (micahgodbolt) [reply apps/vr-tests/src/utilities/steps.ts]: ah, good point. It was working for my single example, so maybe I need to look at using getRTL from fabric utilities

File: apps/vr-tests/.storybook/addons.js [hunk 2]
```diff
@@ -2,4 +2,5 @@
 import '@storybook/addons';
 import '@storybook/addon-options/register';
 import 'storybook-readme/register';
-import '@storybook/addon-actions/register';
\ No newline at end of file
+import '@storybook/addon-actions/register';
+import 'storybook-addon-rtl/register';
```
Comment (dzearing) [inline apps/vr-tests/.storybook/addons.js]: hmmm. is this a new plugin?

File: apps/vr-tests/package.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "style-loader": "^0.19.0",
     "awesome-typescript-loader": "^3.2.3",
     "office-ui-fabric-react-tslint": ">=5.0.0 <6.0.0",
+    "storybook-addon-rtl": "^0.1.1",
```
Comment (dzearing) [inline apps/vr-tests/package.json]: hrm. Doesn't this require a rush generate?

File: apps/vr-tests/.storybook/addons.js (hunk 2 above)
Comment (micahgodbolt) [reply apps/vr-tests/.storybook/addons.js]: I might end up removing this. it wasn't working that well anyway.

File: apps/vr-tests/src/utilities/steps.ts (hunk 1 above)
Comment (micahgodbolt) [reply apps/vr-tests/src/utilities/steps.ts]: Do I need to create a public method on a wrapping component, and then call it via a reference to the component? Or is there an easier way?

**Comment by micahgodbolt** [comment]
closing this up for now. need a better approach

**Pull request closed without merging**